tests/pytests/scenarios/reauth/test_reauth.py

```diff
@@ -0,0 +1,50 @@
+import logging
+import os
+import threading
+import time
+
+import pytest
+
+pytestmark = [
+    pytest.mark.slow_test,
+    pytest.mark.windows_whitelisted,
+]
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope="module")
+def timeout():
+    return os.environ.get("SALT_CI_REAUTH_MASTER_WAIT", 100)
```

Comment:
```suggestion
    return int(os.environ.get("SALT_CI_REAUTH_MASTER_WAIT", 100))
```
The environment variable will always be a string and the code using this fixture likely expects an integer...